### PR TITLE
[OData V2] Document function import calls in batch change sets

### PR DESCRIPTION
**Pull request by Emdee89**
## What Has Changed?

Mentioning the availability of `addFunctionImport(...)` for the OData V2 batch functionality.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [x] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: docs/java/features/odata/type-safe-client-odata-v2.mdx [hunk 1]
```diff
@@ -375,7 +375,9 @@ Refer to the [official OData V2 spec](https://www.odata.org/documentation/odata-
 
 The `service` object offers the method `batch` which allows access to the batch-related methods that help build the batch request.
 
-Multiple single requests (like `createBusinessPartnerAddress`) can be wrapped into so-called changesets.
+Multiple single requests (like `createBusinessPartnerAddress`) can be wrapped into so-called changesets.
```
Comment (artemkovalyov) [inline docs/java/features/odata/type-safe-client-odata-v2.mdx:378]:
```suggestion
Multiple single requests like `createBusinessPartnerAddress` and others can be wrapped into an [OData BATCH  ChangeSet](https://www.odata.org/documentation/odata-version-2-0/batch-processing/). 
```

File: docs/java/features/odata/type-safe-client-odata-v2.mdx [hunk 2]
```diff
@@ -375,7 +375,9 @@ Refer to the [official OData V2 spec](https://www.odata.org/documentation/odata-
 
 The `service` object offers the method `batch` which allows access to the batch-related methods that help build the batch request.
 
-Multiple single requests (like `createBusinessPartnerAddress`) can be wrapped into so-called changesets.
+Multiple single requests (like `createBusinessPartnerAddress`) can be wrapped into so-called changesets. 
+Changesets are meant to wrap transactional requests together.
```
Comment (artemkovalyov) [inline docs/java/features/odata/type-safe-client-odata-v2.mdx:379]:
```suggestion
OData BATCH ChangeSets are meant to wrap transactional requests together for better performance.
```

File: docs/java/features/odata/type-safe-client-odata-v2.mdx [hunk 3]
```diff
@@ -375,7 +375,9 @@ Refer to the [official OData V2 spec](https://www.odata.org/documentation/odata-
...
+Additionally, changesets can also contain function import calls (use the method `addFunctionImport(...)`).
```
Comment (artemkovalyov) [inline docs/java/features/odata/type-safe-client-odata-v2.mdx:380]:
```suggestion
OData Function Import can be also used inside of OData Batch ChangeSets, for that use the method `addFunctionImport(...)`.
```

**Review by artemkovalyov — CHANGES_REQUESTED** [review]
I tried to improve the readability. Take a look.

**Review by artemkovalyov — APPROVED** [review]
LGTM

**Comment by Emdee89** [comment]
@artemkovalyov any idea why the build fails? The vale linter did not find anything in type-safe-client-odata-v2.mdx, which is the only changed file in this PR.

**Comment by artemkovalyov** [comment]
@Emdee89 , my assumption is that with your fork the PR builds based on it. And your fork is not updated with the latest state of the original repos. Might that be the case? I think the errors it produces were fixed alreaady.

**Comment by artemkovalyov** [comment]
Can we try with an updated or new fork to test it?

**Comment by Emdee89** [comment]
@MatKuhr Sure, but the file I am changing is not included in the vale output. So no idea what I need to fix in my file.

**Comment by MatKuhr** [comment]
> Sure, but the file I am changing is not included in the vale output. So no idea what I need to fix in my file.

Nothing. That means the main branch should also have the same issue.

**Pull request merged**